### PR TITLE
fix: support copying PNG to clipboard on Safari

### PR DESCRIPTION
**Pull request by dwelle**
fix https://github.com/excalidraw/excalidraw/issues/4967

Safari doesn't like when the ClipboardItem is created in a different frame than the user-triggered event. https://bugs.webkit.org/show_bug.cgi?id=222262 (via https://web.dev/async-clipboard/). ~~And chrome doesn't support the `Promise` in `ClipboardItem` constructor.~~ EDIT: not anymore, but I'm still handling the case where promise ClipboardItem constructor isn't supported to be safe (future FF may not support it).

### CODE REVIEW

**Review by ad1992 — APPROVED** [review]

File: src/data/index.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export { loadFromBlob } from "./blob";
 export { loadFromJSON, saveAsJSON } from "./json";
 
 export const exportCanvas = async (
-  type: ExportType,
+  type: Omit<ExportType, "backend">,
```
Comment (dwelle) [inline src/data/index.ts:19]: unrelated

**Review by ad1992 — APPROVED** [review]
